src/modules/45-dashboards/pages/home/__tests__/CloneDashboardForm.test.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { act, fireEvent, render, RenderResult, screen, waitFor } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import type { IDashboardFormData } from '@dashboards/types/DashboardTypes'
+import type { StringKeys } from 'framework/strings'
+import * as customDashboardServices from 'services/custom-dashboards'
+import CloneDashboardForm, { CloneDashboardFormProps } from '../CloneDashboardForm'
+
+const defaultProps: CloneDashboardFormProps = {
+  hideModal: jest.fn(),
+  reloadDashboards: jest.fn()
+}
+
+const renderComponent = (props: CloneDashboardFormProps): RenderResult => {
+  return render(
+    <TestWrapper>
+      <CloneDashboardForm {...props} />
+    </TestWrapper>
+  )
+}
+
+const mockEmptyGetFolderResponse: customDashboardServices.GetFolderResponse = {
+  resource: []
+}
+
+describe('CloneDashboardForm', () => {
+  beforeEach(() => {
+    jest
+      .spyOn(customDashboardServices, 'useGetFolder')
+      .mockImplementation(() => ({ data: mockEmptyGetFolderResponse, loading: false } as any))
```

Comment:
Try `mockReturnValue` to simplify a wee bit
```suggestion
      .mockReturnValue({ data: mockEmptyGetFolderResponse, loading: false } as any)
```